### ISSUE TITLE
DEBUG:iqoptionapi.ws.client error

### ISSUE THREAD

**Issue by themow** [Lu-Yi-Hsun/iqoptionapi#93]
From the first there is run normally but When i'm running my code around 45 minutes . How to fix it .
Thank you
There shown like below pic, f
![Screenshot_20190626-172247_LINE](https://user-images.githubusercontent.com/33720845/60172762-85632000-9837-11e9-8bd2-45fa96afbc8f.jpg)

**Comment by JafferWilson** [comment]
Yes I am facing the same issue now. I have tried looking for it but was not able to detect.

**Comment by Lu-Yi-Hsun** [comment]
Does api work well? Just displays that bug?
Or api stuck?

**Comment by JafferWilson** [comment]
It works but its slow and does not return the result. Even sometimes stop functioning.

**Comment by EVAZA** [comment]
I think there are some subscribe packages that arrive after trades get closed. And result is in those packages. You need to subscribe for them to receive them.

**Comment by Lu-Yi-Hsun** [comment]
@themow  give me your sample code

**Comment by rrfaria** [comment]
the same for me but it still working 

https://github.com/rrfaria/iqoption-bot

**Comment by rrfaria** [comment]
when the api  runs for long time 
the websockets closes and stops all 
even when I seted reconect function

if Iq.check_connect()==False:
            Iq.connect()#if not connect it will reconnect

ERROR:iqoptionapi.ws.client:Connection is already closed.
DEBUG:iqoptionapi.ws.client:Websocket connection closed.


this error should be throw and handle by who is using the code to use reconect funcion because when this error happen it didn't enter in "catch"
And the only way to continue is stop script and start again
